scalableminds/webknossos
# Enable Datastore to watch multiple directories

### Steps to test:
- create second dataset directory next to binaryData, insert it in application.conf `braingames.binary.additionalFolders`
- datasets there should be listed, data loading should work
- for the original directory also.

@normanrz do you think we should do some caching for the lookup determining which directory exists? I’m not sure how cache invalidation would work for this. But right now it feels like we’re introducing many additional disk accesses.

### Issues:
- contributes to #3310 

------
